app/src/main/java/co/touchlab/kampkit/android/BreedViewModel.kt

```diff
@@ -24,7 +24,7 @@ class BreedViewModel : ViewModel(), KoinComponent {
     private val scope = viewModelScope
     private val breedModel: BreedModel = BreedModel()
     private val _breedStateFlow: MutableStateFlow<DataState<ItemDataSummary>> = MutableStateFlow(
-        DataState.Loading
+        DataState.Loading<ItemDataSummary>()
```

Comment:
nitpick: you should be able to remove the  type param right?

Reply:
I thought I was getting a crash without it; I think was a Koin issue. I just added it back in, and not getting the crash, so I'll remove it again.